render/src/main/java/org/codice/imaging/nitf/render/imagerep/ImageRepresentationHandlerFactory.java

```diff
@@ -60,14 +60,14 @@ public static ImageRepresentationHandler forImageSegment(final ImageSegment segm
     }
 
     private static ImageRepresentationHandler getRgbImageRepresentationHandler(final ImageSegment segment) {
-        if (segment.getNumberOfBitsPerPixelPerBand() != Byte.SIZE) {
+        if (segment.getNumberOfBitsPerPixelPerBand() != Byte.SIZE && segment.getNumberOfBitsPerPixelPerBand() != Short.SIZE) {
             // It can be 8, 16 or 32 once we are at CLEVEL 6, but so far we can only do 8 (enough for CLEVEL 3 and 5)
```

Comment:
I think the comment about "so far we can only do 8" is probably out of date too.